source/uwp/Renderer/lib/AdaptiveTextBlock.cpp

```diff
@@ -190,11 +190,8 @@ AdaptiveNamespaceStart
         textBlock->SetText(text);
 
         std::string language;
-        if (!(WindowsIsStringEmpty(m_language.Get())))
-        {
```

Comment:
Without this null check this version of HStringToUTF8 will fail on null pointers causing text blocks that don't have language to fail to serialize. Not sure why you removed this null check, but consider using the other version of this function (and while you're in here for text as well):

        textBlock->SetText(HStringToUTF8(m_text.Get()));
        textBlock->SetLanguage(HStringToUTF8(m_language.Get()));